CMakeLists.txt

```diff
@@ -1,63 +1,46 @@
 PROJECT(pblib)
```

Comment:
If we do `project(pblib LANGUAGES CXX)`, cmake will not look for C compiler and configuring a project will be faster.